versions/3.1.1.md

```diff
@@ -2922,24 +2926,38 @@ components:
 
 #### <a name="discriminatorObject"></a>Discriminator Object
 
-When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object gives a hint about the expected schema of the document. It can be used to aid in serialization, deserialization, and validation.
-
-`discriminator` uses a schema's "name" to automatically map a property value to
-a schema. The schema's "name" is the property name used when declaring the
-schema as a component in an OpenAPI document. For example, the name of the
-schema at `#/components/schemas/Cat` is "Cat". Therefore, when using
-`discriminator`, _inline_ schemas will not be considered because they don't have
-a "name".
+When request bodies or response payloads may be one of a number of different schemas, a Discriminator Object gives a hint about the expected schema of the document.
+This hint can be used to aid in serialization, deserialization, and validation.
+The Discriminator Object does this by implicitly or explicitly associating the possible values of a named property with alternative schemas.
```

Comment:
I like the 3.0.4 text better, and can't spot a semantic difference between the two variants.

```suggestion
When request bodies or response payloads may be one of a number of different schemas, a Discriminator Object can be used to aid in serialization, deserialization, and validation by implicitly or explicitly associating the possible values of a named property with alternative schemas.
```

Reply:
@ralfhandl we previously approved the additional wording (submitted by another contributor) in 3.1.1 and I'd rather not take it out on one person's stylistic preference.  If you don't think this is the correct way to merge them, then we can change it, but I was trying not to delete someone else's work here.